### ISSUE TITLE
Windows 11 Issue after upgrading from 10

### ISSUE THREAD

**Issue by Painbringer07** [ModOrganizer2/modorganizer#1667]
## The problem:
Start Mod Organizer 2 with Oblivion instance.  Lock screen appears and the game never starts.  I have looked at my Task Manager and Oblivion is running in the background, but the game screen never appears.  I am running MO2 as admin, and have excluded the MO2 folder from my Firewall and virus scan.  No errors ever appear, and MO2 never crashes (I have to unlock MO2, and force end the Oblivion process through task manager).  The game starts just fine through Steam, and ever loads xOBSE and my ENB.  I have reinstalled MO2 to see if that would do anything (no change). My mod directory is under Appdata and my install of MO2 is on my M.2 NVME along with Steam and my install of the game. I am using windows 11 that was upgraded from 10, and was using windows 10 that was upgraded from 7.  I have installed a BIOS update as well.  

Hardware: AMD MSI MOBO (w/5600x), MSI Nvidia GPU, Samsung NVME, and Corsair RAM.

Any ideas?  
Thanks

**Comment by LostDragonist** [comment]
You need to post logs if you want any help.

That's what all this stuff was about in the issue template...

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your latest USVFS log, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by arkane149** [comment]
windows 11 is doing this with skyrim se/ae and fallout 4 also. I tried earlier versions of mo2 everything recommended , also tried with no mods activated.  my last logs for mo2 skyrim se/ae https://gist.github.com/arkane149/547e6c549c2ca31b9c037b8ea603892f

**Comment by LostDragonist** [comment]
[2022-02-13 05:02:46.572 D]   . IObit Malware Fighter (antivirus)
[2022-02-13 05:02:46.572 D]   . Webroot SecureAnywhere (antivirus)

There's a high chance one of these programs is causing problems.

**Comment by arkane149** [comment]
removed iobit malware fighter,  (microsoft), no change.

**Comment by Al12rs** [comment]
After removing those programs refresh the Mo2 install by running the installer again and post new logs if the issue persists.

**Comment by Painbringer07** [comment]
Well I found the culprit. I usually install Script Extender plugins directly to the install folder of the game (thinking this may cause less problems). NVAC a Fallout New Vegas anti-crash .dll mod was causing it. Deleted it, everything works now.  I did for good measure switch to a "Portable" instance of MO2, and installed my mods in the same folder as the manager (pulling all the files out of my main OS drive).  I'm not sure if this will be helpful to anyone else, but just letting you know.
